src/table/condensed.scss

```diff
@@ -3,14 +3,17 @@
 @import '../style/index';
 @import './variables';
 
-@mixin iui-tables-condensed {
-  .iui-tables-row {
-    > .iui-tables-cell {
-      padding: $iui-table-density-condensed;
-    }
+@mixin iui-table-condensed {
+  .iui-table-header .iui-cell,
+  .iui-row .iui-cell,
+  + .iui-paginator {
+    min-height: $iui-baseline * 4;
   }
 
-  .iui-tables-paginator > .content {
-    padding: $iui-table-density-condensed;
+  + .iui-paginator .iui-select-button {
```

Comment:
I removed all these rules and it looks the same.

Reply:
If you set focus on the select it's more obvious.  Here is how it looks now:
![Screen Shot 2021-06-08 at 5 09 32 PM](https://user-images.githubusercontent.com/849817/121257954-4d1a4800-c87c-11eb-8cfa-87021c766999.png)

And if you remove the rules:
![Screen Shot 2021-06-08 at 5 10 38 PM](https://user-images.githubusercontent.com/849817/121258078-76d36f00-c87c-11eb-9788-5034c7be5f36.png)

All these rules can be removed once #97 passes.  I've added a`.iui-small` class to the select component that does nothing until #97 is merged.

Reply:
Fixed